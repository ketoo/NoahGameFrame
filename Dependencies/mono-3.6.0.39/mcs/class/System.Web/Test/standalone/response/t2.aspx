<% @Page Language="C#" %>
<%
	//
	// This test sets buffering to false and writes
	// which should trigger a chunked response
	//
	
	Response.Buffer = false;
	Response.Output.Write ("hello");
%>