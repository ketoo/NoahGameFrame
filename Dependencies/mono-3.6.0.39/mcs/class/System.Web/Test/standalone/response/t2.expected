5
hello
0

